tests/run_time/CMakeLists.txt

```diff
@@ -22,12 +22,12 @@
 
 # # # # sol2 tests - runtime tests
 
-file(GLOB sources
-	LIST_DIRECTORIES FALSE
-	CONFIGURE_DEPENDS
-	source/*.cpp)
-
-if (WAY_TOO_BUSTED_RIGHT_NOW)
+# Many are broken, so we list the working ones manually for now.
+set(sources
+	source/is_integer.cpp

```

Comment:
I didn't test which other files might also still work. I can of course add other files too, if you tell which ones should work.

Reply:
I think this should not be touched and I should focus on fixing the tests in totality..